functional-test/src/test/groovy/org/rundeck/util/common/execution/ExecutionUtils.groovy

```diff
@@ -40,6 +40,18 @@ class ExecutionUtils {
             }
         }
 
+        /**
+         * Returns a boolean that verifies if the executions where executed sequentially,
+         * meaning the first execution date-ended is lesser than the second's start-date.
+         *
+         * @param executions is the list of executions to verify sorted from first executed to last executed.
+         * @return Boolean
+         */
+        static final haveExecutedSequentially(List<Execution> executions) {
+            executions.collate(2, 1, false)
+                    .every { pair -> pair[1]["date-started"]["unixtime"] >= pair[0]["date-ended"]["unixtime"] }
```

Comment:
nit:
Only if you could say:
`pair[1].enderBeforeOrAt(pair[0])`
Leaving the knowledge of the start and end dates to the `Execution` class where it belongs ;-)